sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-2021-2022.bpmn

```diff
@@ -1451,7 +1451,7 @@ or (calculatedDataPartnerFedralProvincialPDReceiptCost &gt; 0)</bpmn:conditionEx
     <bpmn:intermediateThrowEvent id="Event_0arma0u" name="awardEligibilityBCAG2Year">
```

Comment:
Are we considering changing the name of the event? as the existing one says awardEligibilityBCAG2Year and now we are considering from less than 2 years too? I know this will cause cascading effects renaming in couple of places and the e2e change. I would like to ask on the devs perspective on this @andrepestana-aot @andrewsignori-aot @lewischen-aot @dheepak-aot

Reply:
Renaming the var name is not hard in VSCode. It  can be done at once. We should take a look at the comments and other related places.

Reply:
I see that award eligibility for less than two years is calculated in `awardEligibilityBCAGYear` and `awardEligibilityBCAG2Year` is for two or more years. I'm good with the current naming approaches.

Reply:
Should I rename `awardEligibilityBCAG2Year` to `awardEligibilityBCAGAbove60Weeks` and `awardEligibilityBCAG` to `awardEligibilityBCAG12To59Weeks`.

Reply:
IMO Renaming the variable is meaningful and needed, but the question is should it be done in same PR?
I would recommend doing the renaming of variable in a separate PR at least not polluting this one.

Reply:
I agree to have it done in a separate PR but I still struggling to see the benefits of renaming it.
Maybe I am missing something but please let me know which of the below is the point.
1. The changes in the conditions make the current names no longer have meaning.
2. It would be good to have them renamed but the way that they are right now still preserves the meaning.

If it is about item 1 I would like to understand it better.
If it is about item 2 and multiple files will be affected I would recommend not proceeding with it at all.

Reply:
I'm okay with renaming it. I agree with @andrewsignori-aot and @dheepak-aot. If we want to rename it, there will be many places that need the change. It would be better to rename it in a separate ticket.

Reply:
Thank You everyone for your inputs. Will proceed with renaming **(if needed)** in a separate PR.

Reply:
@andrewsignori-aot 

I believe in 1 because BCAG1 and BCAG2 are eventually using different DMN(which is also named by year) to calculate the value, but naming them with year doesn't seem to be accurate to me as other criteria's are involved in it.